mreg/api/v1/tests.py

```diff
@@ -450,6 +450,165 @@ def test_mx_zone_autoupdate_delete(self):
         self.assertTrue(self.zone.updated)
 
 
+class APIPtrOverrideTestcase(MregAPITestCase):
+    """Test PtrOverride records."""
+
+    def setUp(self):
+        super().setUp()
+        self.host_data = {'name': 'ns1.example.org',
+                          'contact': 'mail@example.org'}
+        self.client.post('/hosts/', self.host_data)
+        self.host = Host.objects.get(name=self.host_data['name'])
+
+        self.ptr_override_data = {'host': self.host.id,
+                                  'ipaddress': '10.0.0.2'}
+
+        self.ptr_override_ipv6_data = {'host': self.host.id,
+                                       'ipaddress': '2001:db8::beef'}
+
+        self.client.post('/ptroverrides/', self.ptr_override_data)
+        self.ptr_override = PtrOverride.objects.get(ipaddress=self.ptr_override_data['ipaddress'])
+        self.client.post('/ptroverrides/', self.ptr_override_ipv6_data)
+        self.ptr_ipv6_override = PtrOverride.objects.get(ipaddress=self.ptr_override_ipv6_data['ipaddress'])
+                                       
+        self.ptr_override_patch_data = {'host': self.host.id,
+                                        'ipaddress': '10.0.0.3'}
+        
+        self.ptr_override_ipv6_patch_data = {'host': self.host.id,
+                                             'ipaddress': '2001:db8::feed'}
+     
```

Comment:
Variables only used in function should be placed there instead to keep `setUp` smaller.